app/assets/javascripts/components/AdminPanel/lessons_page.es6.jsx

```diff
@@ -0,0 +1,32 @@
+class LessonsPage extends React.Component {
+
+  static get propTypes() {
+    return {
+      lessons: React.PropTypes.array,
+    };
+  }
+
+  renderLesson(lesson) {
+    return (
+      <LessonItem lesson={lesson} />
+    );
+  }
+
+  renderLessons() {
+    return this.props.lessons.map((lesson) => this.renderLesson(lesson));
+  }
+
+  render () {
```

Comment:
Can you follow the format for the rest of the pages so far? So outside div should have classname page-wrapper, then the div with all the content should be content-wrapper. Also, you shouldn't have to wrap the adminheader inside another div.